src/server/scripts/Outland/HellfireCitadel/ShatteredHalls/instance_shattered_halls.cpp

```diff
@@ -81,12 +149,31 @@ class instance_shattered_halls : public InstanceMapScript
                             HandleGameObject(nethekurseDoor1GUID, false);
                             HandleGameObject(nethekurseDoor2GUID, false);
                         }
+                        else if (state == DONE)
+                        {
+                            HandleGameObject(nethekurseDoor1GUID, true);
+                            HandleGameObject(nethekurseDoor2GUID, true);
+                        }
```

Comment:
Meh, ignore that, I'll convert the thing into doordata, as the door is supposed to be closed on beginning, forcing you to go through the sewers